contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -240,19 +247,19 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
         // Get the first token to mint
         uint256 start = range.start + range.minted;
 
-        // Pre-mint the range to the seller
+        // Pre-mint the range
         uint256 tokenId;
-        address seller = owner();
         for (uint256 i = 0; i < _amount; i++) {
             tokenId = start + i;
-            emit Transfer(address(0), seller, tokenId);
+
+            emit Transfer(address(0), range.owner, tokenId);
```

Comment:
Please store `range.owner` to memory variable, otherwise it's reading from the storage in every iteration.

Reply:
done